app/engintron.php

```diff
@@ -141,7 +141,7 @@ function execute($act) {
             break;
 
         case "mysql_restart":
-            $command = shell_exec("/scripts/restartsrv_mysql");
```

Comment:
/scripts/restartsrv_mysql is the official script distributed by cPanel to restart mySQL